test/e2e/util.go

```diff
@@ -58,6 +58,13 @@ const (
 
 	// How often to poll pods.
 	podPoll = 5 * time.Second
+
+	// service accounts are provisioned after namespace creation
+	// a service account is required to support pod creation in a namespace as part of admission control
+	serviceAccountProvisionTimeout = 2 * time.Minute
```

Comment:
I did not hit this timeout, but after running e2e many times, I aimed high first.